tests/test_bsfs.c

```diff
@@ -223,7 +231,20 @@ START_TEST(test_unlink) {
   strcat(path, "/bla");
   ck_assert_int_eq(bsfs_mknod(tmp_fs, path, S_IFREG), 0);
   ck_assert_int_eq(bsfs_unlink(tmp_fs, path), 0);
-  // TODO: Make sure file doesn't exist!
+  bs_file_t file;
+  ck_assert_int_eq(bsfs_open(tmp_fs, path, &file), -2);
```

Comment:
Why not use constants here?
```suggestion
  ck_assert_int_eq(bsfs_open(tmp_fs, path, &file), -ENOENT);
```

Reply:
Resolved in https://github.com/ThatFatPat/BSFS/pull/37/commits/9b8fbd388936909eed5bd95d37a93b8b4711f1f3